packages/salesforcedx-vscode-core/package.json

```diff
@@ -61,6 +62,7 @@
     "mocha-multi-reporters": "^1.1.7",
     "mock-spawn": "0.2.6",
     "nyc": "^13",
+    "proxyquire": "2.1.3",
```

Comment:
add proxyquire as a dev dep to the core project to enable unit testing outside the integration test flow.  We are using this in a few packages already, but didn't have any references in core.